apps/condo/pages/ticket/[id]/pdf.tsx

```diff
@@ -124,7 +124,7 @@ const PdfView = () => {
     }
 
     const TicketCreationDate = getTicketCreateMessage(intl, ticket)
-    const ticketAddress = get(ticket, ['property', 'address'])
+    const ticketAddress = get(ticket, ['propertyAddress'])
```

Comment:
it's better to use `property.address` here and user `propertyAddress` as fallback in case of `property === null`. It means the property is DELETED. And we ned to show it to user

Reply:
changed